### DISCUSSION TITLE
Login to Rimusic

### DISCUSSION THREAD

**Discussion by Aravind-art** [fast4x/RiMusic#4177] [Q&A]
Hi, if I login to the app, does it get me banned from Google ? My gmail is being preserved over 15 years, I don't want to loose data.

**Comment by MeiRos** [comment]
Propably not. But if you want to be sure or just don't want to login, use ViTune. It's still working without login.
Hopefully things get solved soon. It would be great to use RiMusic without login again. I don't know why ViTune can and many others can't play without login.

**Comment by DeadSecTr** [comment]
Or try Harmony Music
